doc/main/tbb_userguide/Guiding_Task_Scheduler_Execution.rst

```diff
@@ -61,7 +61,8 @@ assign a NUMA node identifier to the ``task_arena::constraints::numa_id`` field.
 
 The processors with `Intel® Hybrid Technology <https://www.intel.com/content/www/us/en/products/docs/processors/core/core-processors-with-hybrid-technology-brief.html>`_
 contain several core types, each is suited for different purposes.
-For example, some applications may improve their performance by preferring execution on the most performant cores.
+In most cases, hybrid systems don't require special API usage but in special cases, the performance
+may be improved by setting the preferred core type.
 To set execution preference, assign specific core type identifier to the ``task_arena::constraints::core_type`` field.
```

Comment:
Don't you think, that "execution preference" sounds too vague?

Reply:
I guess it can be changed to **"To set preferred core type for the execution"** to continue the sentence above.

Reply:
```suggestion
To set preferred core type for the execution, assign specific core type identifier to the ``task_arena::constraints::core_type`` field.
```